package_creator.py

```diff
@@ -23,6 +26,59 @@
 IMAGE_PREFIX = 'data:image/png;base64,'
 
 
+def write_yaml_with_docker(output_path, yml_text, yml_data, script_obj):
```

Comment:
can we add test for that one?

Reply:
We don't have any tests for this script. We need to refactor it. Let's do this as part of demisto-sdk.